### ISSUE TITLE
`clock` is depricated and doesn't exist in python 3.8

### ISSUE THREAD

**Issue by mtst99** [pedroCabrera/PyFlow#79]
`clock` is deprecated and doesn't exist in Python 3.8 so I get this error.

```
C:\Users\mtst\Documents\softwares\PyFlow-master
λ pyflow
Traceback (most recent call last):
  File "C:\Users\mtst\Documents\softwares\PyFlow-master\pyflow.py", line 17, in <module>
    from PyFlow.App import PyFlow
  File "C:\Users\mtst\Documents\softwares\PyFlow-master\PyFlow\App.py", line 23, in <module>
    from time import clock
ImportError: cannot import name 'clock' from 'time' (unknown location)
```
The same issue discussed here: https://github.com/PyTables/PyTables/issues/744

**Comment by IlgarLunin** [comment]
3.8 is not yet supported. Thank you for submission, will look into it later

**Comment by jaitaiwan** [comment]
Relates to this issue: https://github.com/wonderworks-software/PyFlow/issues/73

**Comment by vocx-fc** [comment]
FreeCAD is developing an add-on using PyFlow.

When this issue popped up, it seems the solution is simply replacing `time.clock()` with `time.process_time()`. It doesn't seem like a difficult change, and it seems `process_time` is available since Python 3.3.

https://stackoverflow.com/a/58569410

**Comment by IlgarLunin** [comment]
Closed in a693d95adc764e0a8083662d54573dbb28c3eeb1
